io/eolearn/io/local_io.py

```diff
@@ -39,18 +42,39 @@ def execute(self, eopatch, *, filename):
         feature_type, feature_name = next(self.feature(eopatch))
         array = eopatch[feature_type][feature_name]
 
-        if self.band_count == 1:
-            array = array[..., 0]
-
         dst_shape = array.shape
-        dst_transform = rasterio.transform.from_bounds(*eopatch.bbox, width=dst_shape[1], height=dst_shape[0])
+        filename_list = []
+        print(self.date_count)
+
+        if str(feature_type) in ['FeatureType.DATA','FeatureType.MASK']:
```

Comment:
Instead of this you can simply check:
```Python
if feature_type in [FeatureType.DATA, FeatureType.MASK]:
```
(Note that `FeatureType` is an `Enum` class therefore you don't have to cast it into string)